### ISSUE TITLE
A fixer for remove [var_dump(), print_r(), exit(), die()] ?

### ISSUE THREAD

**Issue by gregoirepaqueron** [FriendsOfPHP/PHP-CS-Fixer#766]
would you be interested in a fixer that would remove these functions?

Some developers (like me) use theses functions to debug their code, and sometimes, forget to remove them. A fixer to clean theses functions would be great to ensure there will be no problems in front.

I need to know if this fixer would interest you before starting to code it.

**Comment by drAlberT** [comment]
exit() and die() are considered major issue by sensio insight .. so it may be considered IMHO .. 

How you do you think to handle their removal?

**Comment by SpacePossum** [comment]
It would change code behavior in a massive way, so I would vote -1 for removal. It is true that the Fixer sometimes changes code behavior, but IMHO this is too much out of scope.
